### ISSUE TITLE
Abnormal circulating Survival Motor Neuron (SMN) protein concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5986]
**Preferred term label:** Abnormal circulating Survival Motor Neuron (SMN) protein concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of SMN protein in the blood is outside the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://pubmed.ncbi.nlm.nih.gov/32511765/

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating SMN protein concentration

**Synonyms** Concentration of SMN protein in the blood is below the normal range.

**Definition (free text, please give PubMed ID)** 

**Parent term (use hpo.jax.org/app)** Abnormal circulating Survival Motor Neuron (SMN) protein concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating SMN protein concentration

**Synonyms** Concentration of SMN protein in the blood is above the normal range.

**Definition (free text, please give PubMed ID)** 

**Parent term (use hpo.jax.org/app)** Abnormal circulating Survival Motor Neuron (SMN) protein concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
There is no accepted normal range for serum SMN. Therefore, it is not possible to say in an individual patient whether the concentration is abnormal -- therefore, this is not (yet) a good HPO term. The published studies involve cohorts.
